cli/determined_cli/experiment.py

```diff
@@ -307,10 +308,12 @@ def local_experiment(args: Namespace) -> None:
     # directory to model_def.
     #
     # Reference: https://docs.python.org/3/library/sys.html#sys.path
+
+    with experimental._local_execution_manager(args.model_def.resolve()):
+        trial_class = load.load_trial_implementation(experiment_config["entrypoint"])
+
     sys.path = [""] + sys.path
-    experimental.test_one_batch(
-        args.model_def.resolve(), trial_class=None, config=experiment_config
-    )
+    experimental.test_one_batch(trial_class=trial_class, config=experiment_config)
```

Comment:
blocking: i believe that the `_local_execution_manager` is going to be necessary for `test_one_batch`, because all of the plugin function implementations are conditioned on the assumption that the local cwd == the context directory. For example, if they have a `open("metadata.json")` call somewhere in the `train_batch`, `evaluate_batch`, etc. implementation, this test will fail if the cwd they're executing from is not the context directory.

Reply:
Unify all code paths of local mode to use `_local_execution_manager`.